cms/static/sass/views/_container.scss

```diff
@@ -331,3 +331,112 @@
     }
   }
 }
+
+
+.move-xblock-modal {
+
+  button {
+    background: transparent;
+    border-color: transparent;
+    padding: 0;
+    border: none;
+  }
+
+  .breadcrumb-container {
+    margin-bottom: ($baseline/4);
+    border: 1px solid #D9D9D9;
+    padding: ($baseline/2);
+    background: #F2F8FB;
+
+    .breadcrumbs {
+      .bc-container {
+        @include font-size(14);
+
+        .breadcrumb-fa-icon {
+          @include padding(0, ($baseline/4), 0, ($baseline/4));
```

Comment:
This doesn't need to use ``padding`` as the left and right padding are identical. You can just use:

    padding: 0 ($baseline/4);